packages/gensx-anthropic/package.json

```
@@ -0,0 +1,57 @@
+{
+  "name": "@gensx/anthropic",
+  "version": "0.1.7",
+  "description": "Anthropic integration for GenSX",
+  "type": "module",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "engines": {
+    "node": ">=18.0.0"
```

```suggestion
    "node": ">=18.0.0",
    "pnpm": ">=9.0.0"
```

We need to go update this everywhere.

---

updated. want me to go ahead and update this on all the other packages?

---

sure if you feel like it. Not high priority though